index.html

```diff
@@ -43,13 +43,17 @@ <h1 style="color: white"><i class="fa fa-map-marker fa-2x" aria-hidden="true"></
 			<div class="dropdown">
   				<button onclick="myFunction()" class="dropbtn" id="button2">Date</button>
 			</div>
-			</div>	
+			</div>
 			<br>
 			<div class="col-2-sm">
 			<div class="dropdown">
   				<button onclick="myFunction()" class="dropbtn" id="button3">Zip Code</button>
 			</div>
 			</div>
+			<br>
+			<div class="col-2-sm">
+			<div class="dropdown">
+			<input type="button" value="Reset Zoom" id="button4" onClick="window.location.reload()">
```

Comment:
Deepaul, is it necessary to have the onClick attribute if you already have a callback registered for id="button4" in the controller.js (line 373)?